src/Explorer/Graph/GraphExplorerComponent/GraphData.ts

```diff
@@ -413,14 +413,14 @@ export class GraphData<V extends GremlinVertex, E extends GremlinEdge> {
    * @param node
    * @param prop
    */
-  public static getNodePropValue(node: D3Node, prop: string): string | number | boolean {
+  public static getNodePropValue(node: D3Node, prop: string): undefined | string | number | boolean {
     if (node.hasOwnProperty(prop)) {
       return (node as any)[prop];
     }
 
     // This is DocDB specific
-    if (node.hasOwnProperty("properties") && node.properties.hasOwnProperty(prop)) {
-      return node.properties[prop][0]["value"];
+    if (node.hasOwnProperty("properties") && node.properties!.hasOwnProperty(prop)) {
+      return node.properties![prop][0]["value"];
```

Comment:
This should work. 
```suggestion
      return node.properties?prop[0]?value;
```

Reply:
![image](https://user-images.githubusercontent.com/64865559/103957040-6ffad180-510f-11eb-8df5-ba853c339304.png)

Reply:
I chenged to "return node.properties?.prop[0]?.value;"